### ISSUE TITLE
SC 1.4.10 - Implementation Report - PDF

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#408]
_From @kerstinp on June 3, 2018 17:42_

Hi all,

some comments and questions on the implementation report: [https://www.w3.org/WAI/WCAG21/implementation-report/implementation?implementation_id=181](https://www.w3.org/WAI/WCAG21/implementation-report/implementation?implementation_id=181).

Unfortunately the version which was used for the testing the Reflow-SC and the original version differ. 

It seems that this one is the (not tagged) original version of the tested PDF, or?: [https://wwwimages2.adobe.com/content/dam/acom/en/security/pdfs/adb_security-culture-wp.pdf](https://wwwimages2.adobe.com/content/dam/acom/en/security/pdfs/adb_security-culture-wp.pdf) 

If so, the original version contains an image on the second page which is not present in the tagged version used for the implementation testing I would think that when testing wether the requirements of a new SC are implementable the versions should be the same without deleting any content, or?

My second issue is that the implementation report gives no information about the testing procedure and the tool which was used for the test. This is imo especially important because there is one reflow according to the tag order and the Adobe reflow. People might think that the Adobe Reflow was tested and that it is no problem when the address information (left side on page ) are not visible in Adobe reflow mode (tested with Adobe Reader last german version and Acrobat Pro DC also last german version). So for SC Reflow in Adobe reflow I would say it fails. I therefore find it important to give information in the implementation report on which reflow with which tool was used for testing the new Reflow-SC .

The implementation report is as far as I know especially for the new SC and not for the SC which are already in WCAG 2.0. But some aspects for two of the "old" SC for the tested PDF on [http://awkawk.github.io/ADB_Security%20Culture%20WP_FINAL.pdf](http://awkawk.github.io/ADB_Security%20Culture%20WP_FINAL.pdf). (Edit kerstinp: the PDF which is now behind the link was modified )

I would think that it is not a pass for SC 1.3.1 – out of several reasons:

- Text on the left side on page 3 (from Adobe Engineering to acrosss the company). I would say that this is a heading and a list with four entries. But just P-Tags have been used.
- For the content above the text "Figure 1: …" (4 icons with text explainations) on page 3 also just P-Tags have been used. I would think that this is also a list with four entries.
- For the caption "Figure 1: ASSET Software Security Certification Program" not the Caption-Tag was used but also just a P-Tag.
- And there is one small mistake on page 6. The paragraph in the text starts with "In addition,…". But according to the P-Tag the paragraph starts with "step ahead…"

Please find screenshots with the tree and marked text attached.

One other thing is that the content above the caption is a combination of four images (four Icons) and a text for every icon – so it is not a figure, but the Caption says: "Figure 1". I would think that this is not a pass for SC 2.4.6. It would be a pass for 2.4.6 when icons and texts would be an image (which of course would fail 1.4.5 but that's not the point).

But these are not the main points of this github-issue. My main points are that the version of the PDF with the implementation and the original version differ and that there is no documentation on which reflow was tested with which tool.

![no-list-01](https://user-images.githubusercontent.com/6586455/40889394-22b2387c-6766-11e8-8927-10b23accc348.PNG)

![no-list-02](https://user-images.githubusercontent.com/6586455/40889400-3447a108-6766-11e8-8770-591f6efafff6.PNG)

![p-tags](https://user-images.githubusercontent.com/6586455/40889402-39e68002-6766-11e8-87bc-d73a9cad8440.PNG)

_Copied from original issue: w3c/wcag21#949_

**Comment by michael-n-cooper** [comment]
_From @awkawk on June 4, 2018 17:52_

@kerstinp The testing was done using Adobe Reader on Mac and Windows, perhaps more. I've just tried the file on the Mac using Reader and reflow is working as expected, including the address information.

The file that was used for Implementation testing does differ from the published file on Adobe's site because we were not able to get access to all of the assets that were needed to make the fixes so we removed that image and the implementation results are based on that document.

> I would think that it is not a pass for SC 1.3.1 – out of several reasons:
> 
> Text on the left side on page 3 (from Adobe Engineering to acrosss the company). I would say that this is a heading and a list with four entries. But just P-Tags have been used.
> For the content above the text "Figure 1: …" (4 icons with text explainations) on page 3 also just P-Tags have been used. I would think that this is also a list with four entries.
I think that these are interpretation issues. These could be interpreted as a list or as paragraph text and we chose the latter. 

> For the caption "Figure 1: ASSET Software Security Certification Program" not the Caption-Tag was used but also just a P-Tag.
I made this change in the file now available at the same address.

> And there is one small mistake on page 6. The paragraph in the text starts with "In addition,…". But according to the P-Tag the paragraph starts with "step ahead…"
Agreed, fixed.

**Comment by michael-n-cooper** [comment]
_From @kerstinp on June 4, 2018 19:20_

Hi @awkawk,

please find attached the screenshots from the last page - Adobe reflow in Adobe Reader and Adobe reflow in Acrobat Pro DC. The address information is not visible. Which means that the testing results are not the same. 

Could you explain what

> we were not able to get access to all of the assets that were needed to make the fixes 

exactly means? 

I would think that because content was removed it is not really suitable as a proof for the implementation of the new SC Reflow. 

![address-adobe-reader](https://user-images.githubusercontent.com/6586455/40935833-9c393f96-6839-11e8-9e29-144c59d440f2.PNG)

![adress-acrobat-pro-dc](https://user-images.githubusercontent.com/6586455/40935842-a6d6d63e-6839-11e8-8e01-c13e496bf5cf.PNG)

**Comment by michael-n-cooper** [comment]
_From @mraccess77 on June 4, 2018 19:23_

I see the address in Acrobat Pro with reflow.
![image](https://user-images.githubusercontent.com/5640755/40936969-27674dda-680b-11e8-9e08-465e30443dcb.png)

**Comment by michael-n-cooper** [comment]
_From @awkawk on June 4, 2018 19:27_

It means that the PDF was an InDesign-created document and the assets that were linked weren't all available so we couldn't work with them, and in order to conform we needed them (I believe that text contrast was an issue, but probably more). Since we couldn't get them we removed the image that was repeating text content.

I'm not sure why you aren't seeing the address, but that sounds like a possible bug somewhere and we will need to see if we can figure that out.

The conformance claim is not for the original document, it is for the modified one. Other implementations made modifications in order to demonstrate that they could meet WCAG 2.1 success criteria, it shouldn't be surprising that a PDF might also need to.

**Comment by michael-n-cooper** [comment]
_From @kerstinp on June 4, 2018 19:58_

@awkawk 

I will try to find others for having a look also and for screenshots.

> Other implementations made modifications in order to demonstrate that they could meet WCAG 2.1 success criteria, it shouldn't be surprising that a PDF might also need to.

When it is necessary to remove content like the dropped in order to meet WCAG 2.1 it's a serious problem. I think that PDF should not be in scope of the Reflow SC.

**Comment by michael-n-cooper** [comment]
_From @kerstinp on June 4, 2018 23:10_

@awkawk

I saw that with the new version of the PDF also some modifications for the address information has been done, like deleting an Formular-XObject. 

After these modifications now the address information is visible in Adobe Reflow here.

Unfortunately not all parts of the discussion and also not the mentioned problem with the invisible information is understandable without the originally tested PDF used for the implementation.

I have of course that one here but for letting other people do checks with the original one to see wether the address information was visible or not the originally tested PDF should be also online.

I think that the whole case with different results including both PDF should be documented in the Implementation Report.

**Comment by fstrr** [comment]
Closing as this was labelled as "Inactive - close soon" in 2019.
